utils/utils.jsx

```diff
@@ -626,7 +626,7 @@ export function applyTheme(theme) {
         changeCss('.app__body .bg--white, .app__body .system-notice, .app__body .channel-header__info .channel-header__description:before, .app__body .app__content, .app__body .markdown__table, .app__body .markdown__table tbody tr, .app__body .suggestion-list__content, .app__body .modal .modal-footer, .app__body .post.post--compact .post-image__column, .app__body .suggestion-list__divider > span, .app__body .status-wrapper .status, .app__body .alert.alert-transparent, .app__body .post-image__column', 'background:' + theme.centerChannelBg);
         changeCss('#post-list .post-list-holder-by-time, .app__body .post .dropdown-menu a', 'background:' + theme.centerChannelBg);
         changeCss('#post-create, .app__body .emoji-picker__preview', 'background:' + theme.centerChannelBg);
-        changeCss('.app__body .modal-body, .app__body .date-separator .separator__text, .app__body .new-separator .separator__text', 'background:' + theme.centerChannelBg);
+        changeCss('.app__body .modal-content, .app__body .date-separator .separator__text, .app__body .new-separator .separator__text', 'background:' + theme.centerChannelBg);
```

Comment:
Looks good to me, but just to confirm that this is reverting this change you made previously. Is that going to cause a regression there?

https://github.com/mattermost/mattermost-webapp/pull/2234/files#diff-3fadf2896914d755b6f1baa150454ef1L624

Reply:
I think should be fine. Not sure why I did that, but I checked the revoke sesion modal, and it seems to be working fine (among a bunch of other ones).